### PR TITLE
Fix importing Fastfile for all tools

### PR DESCRIPTION
**Pull request by asfalcone**
All tools previously imported the fastlane Fastfile from git. Since we're in a monorepo now, we can just import the file directly :tada:

### CODE REVIEW

**Comment by i2amsam** [comment]
LGTM

**Comment by mfurtak** [comment]
:ship: it

File: supply/fastlane/Fastfile [hunk 1]
```diff
@@ -1,2 +1,2 @@
 # Fetch and use the latest Fastfile from the fastlane main repository
-import_from_git(url: "https://github.com/fastlane/fastlane/")
+import("../../fastlane/fastlane/Fastfile")
```
Comment (i2amsam) [inline supply/fastlane/Fastfile]: Missing final newline

**Comment by KrauseFx** [comment]
:+1:

**Pull request merged**